ntbs-service/Services/AlertService.cs

```diff
@@ -20,6 +21,7 @@ public interface IAlertService
         Task DismissAlertAsync(int alertId, string userId);
         Task AutoDismissAlertAsync<T>(Notification notification) where T : Alert;
         Task DismissMatchingAlertAsync<T>(int notificationId, string auditUsername = AuditService.AuditUserSystem) where T : Alert;
+        Task DismissAllOpenAlerts(int notificationId);
```

Comment:
I think `DismissAllOpenAlertsForNotification` would be a slightly clearer name here